h/models/token.py

```diff
@@ -71,9 +71,4 @@ def get_dev_token_by_userid(cls, session, userid):
                 .first())
 
     def regenerate(self):
-        self.value = self.prefix + _token()
-
-
-def _token():
-    """Return a random string suitable for use in an API token."""
-    return binascii.hexlify(os.urandom(16))
+        self.value = self.prefix + security.token_urlsafe()
```

Comment:
Although it isn't the nicest way of testing, it might make sense to add one new test that checks that we're using `security.token_urlsafe()`, which basically tests that whatever we're using, it will be safe to put it in the URL.